### PR TITLE
UI/request form labels

### PR DESCRIPTION
**Pull request by luisgov**
This PR removes labels from choice inputs (radio buttons) and style the legend the same way as input text fields.

### CODE REVIEW

File: templates/components/text_input.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {% macro TextInput(field, placeholder='') -%}
   <div class='usa-input {% if errors %}usa-input--error{% endif %}'>
-    <label for={{field.name}}>
```
Comment (andrewdds) [inline templates/components/text_input.html]: Im not sure CSS is sufficient for fixing this. The `field.description` no longer has any accessible association with the input. By being inside the label, it had this association. We'll need to add some aria attributes to create that association, `labeledby` or some such thing. Same for the error messages, now that I think of it.

Also, did you check what happens to the icon positioning? It was absolutely positioned relative to the parent label, which it no longer has as a parent. I imagine it lost its positioning context and might be positioned somewhere weird now.

**Comment by andrewdds** [comment]
Also @luisgov could your rebase this? I'm having trouble running it since its so far behind master

File: styles/elements/_inputs.scss [hunk 2]
```diff
@@ -100,9 +100,17 @@
...
+      label {
+        font-weight: $font-bold;
+      }
+
       .icon {
         vertical-align: middle;
       }
+
+  .usa-input__help {
+    font-weight: $font-bold;
```
Comment (andrewdds) [inline styles/elements/_inputs.scss]: Shouldn't the help text be normal weight?

**Comment by andrewdds** [comment]
I think a lot of these CSS changes can be avoided if we simply strip the HTML from the `{{ field.label }}` tag from WTForms. 

Looks like jinja already has a `striptags` filter that might do the trick: `{{ field.label | striptags }}`

File: styles/elements/_inputs.scss (hunk 2 above)
Comment (luisgov) [reply styles/elements/_inputs.scss]: 👍

**Comment by luisgov** [comment]
@andrewcroce, all set.

screen shot 2018-08-09 at 10 26 46

**Pull request merged**